.github/actions/cargo_target_dir_cache/action.yml

```diff
@@ -1,13 +1,16 @@
 name: CARGO_TARGET_DIR Cache
 description: CARGO_TARGET_DIR Cache
+inputs:
+  ockam_dir:
+    description: Ockam Directory
```

Comment:
I find this variable name and description to be a little vague. Can you think of any phrasing that would be more explicit about its purpose, the content that we expect to read from there or write to there, etc.? In the typical case, is this the root of the git clone? Is there a sensible default that could be included?

Reply:
It's the root directory to be cached, this is where the target directory is.

Reply:
Answering my question in-line here is helpful but to make my feedback more explicit: prior to merging, please find a more descriptive variable name and/or a clearer `description` value that expresses that answer **in the code**, because having to run git-blame and find the PR introducing this code in order to understand why it's written that way is a bad experience.

Reply:
@shanesveller I already pushed a commit updating the description https://github.com/build-trust/ockam/blob/d1034e4bb5c027c7db48fd4c95abe99365d60be5/.github/actions/cargo_target_dir_cache/action.yml#L5. I'll be resolving the conversation, please unresolve if you think we can use a better description.